pcweb/pages/docs/gallery.py

```diff
@@ -244,6 +244,16 @@
         "url": "https://moure.dev",
         "source": "https://github.com/mouredev/python-web",
     },
+    {
+        "name": "rx_shout",
+        "difficulty": "Intermediate",
+        "tags": ["Auth", "Database", "Widget", "Forms"],
+        "description": "An embeddable shoutbox widget using Sign-in with Google.",
+        "img": "/gallery/rx_shout.png",
+        "gif": "/gallery/rx_shout.gif",
+        "url": "https://rx-shout.mooo.com",
```

Comment:
I was hoping I didn’t have to deal with self hosted apps so soon

Reply:
Yeah, unfortunately this app needs database and uploaded files. I'd like to move it to the hosting service though, because I'm currently paying for this VPS out of pocket lol.

I'll try to retool it for S3 and a remote Postgres in the next few weeks.